db/DbAbstract.php

```diff
@@ -13,7 +13,7 @@ public static function getTable($name){
     $result = static::getPrefix();
     foreach ($tables as $key => $value) {
       if ($key == $name)
-        $result.= $tables[$value];
+        $result.= $tables[$key];
     }
```

Comment:
Si llegas a tener un arreglo con muchos nombres de tabla y justo pasa que el parámetro `$name` coincide con la primer clave, el for sigue ciclando innecesariamente. Habría que agregarle un `break` dentro del if para que si encuentra la tabla, corte la ejecución del ciclo. En este caso no afecta al rendimiento, pero es una práctica a tener en cuenta para el manejo de arreglos en bucles